Adding missing auto ally presets

Puzzabug opened the pull request:
Certain map were missing presets, I have added them, and standardized preset naming or player count issues on a few maps https://github.com/CnCNet/cncnet-yr-client-package/issues/452

Cheers!

Puzzabug committed:
Adding missing auto ally presets

Certain map were missing presets, I have added them, and standardized preset naming or player count issues on a few maps
https://github.com/CnCNet/cncnet-yr-client-package/issues/452

11EJDE11 commented:
Ran it through an online parser:

error on line 5206:
Property with key 'TeamStartMapping2' already exists in section 'Maps\Yuri's Revenge\xkaliforn'.

error on line 5953:
Property with key 'TeamStartMapping0' already exists in section 'Maps\Yuri's Revenge\xdeath'.

error on line 5954:
Property with key 'TeamStartMapping0Name' already exists in section 'Maps\Yuri's Revenge\xdeath'.

error on line 6693:
Property with key 'TeamStartMapping2' already exists in section 'Maps\Yuri's Revenge\Battle\8_the_eye'.


Also, last I checked, Mayflower Freezes Over has top/bottom and left/right labelled the wrong way around. Could you fix that too?

Puzzabug committed:
Fixing missing auto ally presets

Mayflower Original, Mayflower Frozen Over presets, Mayflower FFA added presets

Puzzabug commented:
Fixed incorrect key names.
Fixed Mayflower Freezes over incorrect naming.
Added Mayflower FFA and added Mayflower Original presets. 

What online parser did you use?

alexp8 commented:
MPMaps.ini is a generated file and shouldn't be manually edited

The ally presets belong in their map file

Puzzabug commented:
> MPMaps.ini is a generated file and shouldn't be manually edited
> 
> The ally presets belong in their map file

Makes sense; how do I fix the map files? I looked through the maps with n++ and world altering editor and didn't find it either.

Edit:

I found 43 maps have the ini keys including TeamStartMapping, but did not see this in most maps. Just a few newer ones, blitz maps and some from Burg

I also found that replacing the ini file with the new one did show the new updates, but obviously I'd want to do this the right way. Let me know!

[Basic]
Name=[4] Distant Quasar
Author=Burg
TeamStartMapping0=A,A,B,B
TeamStartMapping1=A,B,A,B
TeamStartMapping0Name=Left vs Right
TeamStartMapping1Name=Top vs Bottom

alexp8 commented:
let's go ahead and merge this for now - we're just at risk of the values getting removed because the source is the map file

we can revisit if we need to update the map files (preferred way)
